s3.go

```diff
@@ -0,0 +1,57 @@
+package veneur
+
+import (
+	"errors"
+	"io"
+	"path"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+)
+
+const S3Bucket = "stripe-veneur"
+
+const DefaultAWSRegion = "us-west-2"
+const AwsProfile = "veneur-s3-test"
+
+var svc s3iface.S3API
```

Comment:
I'd rather attach it to config than the server, but we can figure that out, yeah.